pipeline/eval/eval.py

```diff
@@ -225,23 +231,43 @@ def main(args_list: Optional[list[str]] = None) -> None:
         target_ref_lines = file.readlines()
     with open(target_file, "r") as file:
         target_lines = file.readlines()
+    with open(source_file, "r") as file:
+        source_lines = file.readlines()
 
     compute_bleu = BLEU(trg_lang=trg)
     compute_chrf = CHRF()
 
-    print("Computing the BLEU score.")
+    logger.info("Computing the BLEU score.")
     bleu_score: BLEUScore = compute_bleu.corpus_score(target_lines, [target_ref_lines])
     bleu_details = json.loads(
         bleu_score.format(signature=compute_bleu.get_signature().format(), is_json=True)
     )
 
-    print("Computing the chrF score.")
+    logger.info("Computing the chrF score.")
     chrf_score: CHRFScore = compute_chrf.corpus_score(target_lines, [target_ref_lines])
     chrf_details = json.loads(
         chrf_score.format(signature=compute_chrf.get_signature().format(), is_json=True)
     )
 
-    data = {
+    # The default comet model.
+    # It should match the model used in https://github.com/mozilla/firefox-translations-models/
+    comet_model_name = "Unbabel/wmt22-comet-da"
+    comet_mode = "cpu" if args.gpus == 0 else "gpu"
+    logger.info(f'Computing the COMET score with "{comet_model_name}" using the {comet_mode}')
+
+    # COMET_MODEL_DIR allows tests to place the model in a data directory
+    comet_checkpoint = comet.download_model(
+        comet_model_name, saving_directory=os.environ.get("COMET_MODEL_DIR")
+    )
+    comet_model = comet.load_from_checkpoint(comet_checkpoint)
+    comet_data = []
+    for source, target, target_ref in zip(source_lines, target_lines, target_ref_lines):
+        comet_data.append({"src": source, "mt": target, "ref": target_ref})
+    comet_results = comet_model.predict(comet_data, gpus=args.gpus)
```

Comment:
You can also control `batch_size` here. The default one might be fine (16 I think), but ideally we should check how it utilizes GPU memory and bump it if it's underutilized.

Reply:
I removed the parameter from it since I figured the defaults would be fine.

> ideally we should check how it utilizes GPU memory

Ah, I haven't used the dashboards yet to do any tuning like this yet.